notebooks/samples/explanations/tf2/ai-explanations-tabular.ipynb

```diff
@@ -1,1483 +1,1432 @@
 {
```

Comment:
<p>Change the sentence to:</p><p> </p><p>You can use the Explainable AI SDK to send explanation requests to your deployed model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
<p>Done.</p>